src/components/skills-quiz/SearchResults.jsx

```diff
@@ -22,26 +23,26 @@ const renderError = (isJobResult) => (
       <FontAwesomeIcon icon={faExclamationTriangle} />
     </div>
     <p>
-      { isJobResult ? JOBS_ERROR_ALERT_MESSAGE : COURSE_ERROR_ALERT_MESSAGE }
+      { isJobResult ? JOBS_ERROR_ALERT_MESSAGE : COURSES_ERROR_ALERT_MESSAGE }
     </p>
   </div>
 );
 
-const hitResultComponent = (isJobResult, currentStep) => {
-  if (!isJobResult) {
-    return SearchCourseCard;
-  }
-  if (currentStep === STEP1) {
-    return SearchJobCard;
-  }
-  return SelectJobCard;
-};
+const renderDialog = () => (
+  <div>
+    <div>
+      <FontAwesomeIcon icon={faExclamationTriangle} />
+    </div>
+    <p>
+      { NO_COURSES_ALERT_MESSAGE }
```

Comment:
Can we please change it to `NO_JOBS_ALERT_MESSAGE`

Reply:
It's actually for courses not jobs. SearchResults doesn't render job cards anymore.